### PR TITLE
Make sure to import os in example and test gaudi opt files

### PR DESCRIPTION
**Pull request by tmadlener**
BEGINRELEASENOTES
- Make sure to have an `import os` in all gaudi options files that use `os` in some form. This becomes necessary after https://github.com/key4hep/k4FWCore/pull/134

ENDRELEASENOTES

### CODE REVIEW

File: doc/starterkit/k4MarlinWrapperCLIC/Readme.md [hunk 1]
```diff
@@ -108,6 +108,7 @@ during [Simulation](#simulation).
 ```bash
 cd CLICPerformance/clicConfig
 
+sed -i `1s/^/import os\n/` clicReconstruction.py
```
Comment (andresailer) [inline doc/starterkit/k4MarlinWrapperCLIC/Readme.md:111]:
```suggestion
sed -i `s/^/import os\n/` clicReconstruction.py
```

**Comment by andresailer** [comment]
There is still a test error about missing `os`
```
11/11 Test #11: clicRec_lcio_mt ..................***Failed    0.64 sec
Traceback (most recent call last):
  File "/cvmfs/sw-nightlies.hsf.org/key4hep/releases/2023-09-12/x86_64-centos7-gcc12.2.0-opt/k4fwcore/d2ee44d38122a401c6ffa3ee6de5f67ddc8c1275=develop-3h7hvo/bin/k4run", line 129, in <module>
    load_file(file)
  File "/cvmfs/sw-nightlies.hsf.org/key4hep/releases/2023-09-12/x86_64-centos7-gcc12.2.0-opt/k4fwcore/d2ee44d38122a401c6ffa3ee6de5f67ddc8c1275=develop-3h7hvo/bin/k4run", line 26, in load_file
    exec(file.read(), {})
  File "<string>", line 176, in <module>
NameError: name 'os' is not defined
```

**Comment by tmadlener** [comment]
I think now I got all of them :crossed_fingers:

**Review by andresailer — APPROVED** [review]

**Comment by andresailer** [comment]
Thanks!

**Pull request merged**